score.py

```diff
@@ -17,11 +17,15 @@ def read_file_to_list(file_in,logger):
             data_list.append(float(line))
```

Comment:
Not related to this change, but I can rarely avoid an opportunity to spread this fact:

There is a one-liner to convert a file that is a list of numbers into a list of floats in Python:

    with open(file_in) as inp:
        data_list = list(map(float, inp)

This works because an open file descriptor when used as an iterator, yields each line in the file in turn.